helm-charts/chatqna/vllm-openvino-values.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+vllm:
+  openvino_enabled: true
+  image:
+    repository: opea/vllm-openvino
+    pullPolicy: IfNotPresent
```

Comment:
Drop the value, it breaks CI testing for `latest` tag (see https://github.com/opea-project/GenAIInfra/pull/587).